SmartTutor/Service/Recommenders/IRecommender.cs

```diff
@@ -0,0 +1,10 @@
+﻿using SmartTutor.ContentModel;
+using System.Collections.Generic;
+
+namespace SmartTutor.Service.Recommenders
+{
+    public interface IRecommender
+    {
+        List<EducationContent> FindEducationalContent(List<SmellType> issues);
```

Comment:
Can you rename the EducationContent to EducationalContent and regenerate the Migrations so that the DB reflects this name.